plans/fix-2821-skill-body-bridge.md

```
@@ -0,0 +1,112 @@
+
+追加で判明した二次不具合: **アシスタントの回答が2回入る**。
+
+## 測定（Claude CLI の生ストリーム）
+
+`claude --output-format stream-json --include-partial-messages` を直接叩いて
+raw イベントを採取。条件を2つ振った（入力 `-p` / stream-json、本文 1,350 字 /
+8,900 字）が結果は同一。対照として Skill を使わないターンも採取した。
+
+実パーサ (`server/agent/stream.ts`) を通した結果:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)